src/amo/reducers/users.js

```diff
@@ -548,12 +548,19 @@ const reducer = (
       if (newState.byID[userId]) {
         const { username } = newState.byID[userId];
 
-        delete newState.byID[userId];
-        delete newState.byUsername[username];
-
-        if (newState.currentUserID === userId) {
-          newState.currentUserID = null;
-        }
+        return {
+          ...newState,
```

Comment:
Do we even need `newState` anymore with this change? Could it just be `state`?

Reply:
Duh. Thanks.